handbook/product/README.md

```diff
@@ -452,7 +452,7 @@ This group maintains the following [Slack channels](https://fleetdm.com/handbook
 |:------------------------------------|:--------------------------------------------------------------------|
 | `#help-product`                     | Mo Zhu                                                              |
```

Comment:
This needs updating

Reply:
Thanks Rachel. Going to follow up w/ a separate PR for these changes so we can get this PR merged in.